ui/packages/catalog-ui-search/src/main/webapp/component/metacard-actions/metacard-actions.view.js

```diff
@@ -15,21 +15,61 @@
 /*global define, setTimeout*/
 const Marionette = require('marionette')
 const _ = require('lodash')
-const $ = require('jquery')
-const template = require('./metacard-actions.hbs')
 const CustomElements = require('../../js/CustomElements.js')
 const store = require('../../js/store.js')
-const MapActions = require('../map-actions/map-actions.view.js')
+import * as React from 'react'
```

Comment:
`import * as React from 'react'` is only needed in tsx files for types, in a javascript file it can just be 
```
import React from 'react'
```